addons/point_of_sale/models/pos_order.py

```diff
@@ -558,6 +558,18 @@ def refund(self):
             'target': 'current',
         }
 
+    def action_receipt_to_customer(self, receipt, cashier_id, client):
```

Comment:
if the order is already server side, cashier_id -> self.user_id and client -> self.partner_id